src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -84,10 +94,34 @@ function ToggleSettingOptionRow({
 }: ToggleSettingOptionRowProps) {
     const styles = useThemeStyles();
 
-    const subTitleView = useMemo(
-        () => <Text style={[styles.textLabel, shouldPlaceSubtitleBelowSwitch ? styles.mt1 : {...styles.mt1, ...styles.mr5}, styles.textSupporting]}>{subtitle}</Text>,
-        [shouldPlaceSubtitleBelowSwitch, subtitle, styles.mr5, styles.mt1, styles.textLabel, styles.textSupporting],
-    );
+    const subtitleHtml = useMemo(() => {
+        if (!subtitle || !shouldParseSubtitle) {
+            return '';
+        }
+        const parser = new ExpensiMark();
+        return parser.replace(subtitle, {shouldEscapeText});
+    }, [subtitle, shouldParseSubtitle, shouldEscapeText]);
+
+    const processedSubtitle = useMemo(() => {
+        let textToWrap = '';
+
+        if (shouldParseSubtitle) {
+            textToWrap = subtitleHtml;
+        }
+
+        return textToWrap ? `<comment><muted-text-label>${textToWrap}</muted-text-label></comment>` : '';
+    }, [shouldParseSubtitle, subtitleHtml]);
+
+    const subTitleView = useMemo(() => {
+        if (!!subtitle && shouldParseSubtitle) {
+            return (
+                <View style={[styles.flexRow, styles.renderHTML, shouldPlaceSubtitleBelowSwitch ? styles.mt1 : {...styles.mt1, ...styles.mr5}]}>
+                    <RenderHTML html={processedSubtitle} />
+                </View>
+            );
+        }
```

Comment:
Why do we need this?

Reply:
I'm sorry but you'll have to be a bit more descriptive than that 😄 - Why do we need to do this?